docs/installation/debian.rst

```diff
@@ -42,7 +42,7 @@ Install from apt.mopidy.com
 
 #. Add the archive's GPG key::
 
-       wget -q -O - https://apt.mopidy.com/mopidy.gpg | sudo apt-key add -
+       sudo wget -q -O /etc/apt/trusted.gpg.d/mopidy.gpg https://apt.mopidy.com/mopidy.gpg
```

Comment:
Third party keys should not be placed here. See https://wiki.debian.org/DebianRepository/UseThirdParty:

> The key MUST NOT be placed in /etc/apt/trusted.gpg.d or loaded by apt-key add.

As far as I've understood, this is because keys placed there would get the rights to sign any package on your system.

It should rather be placed in `/usr/local/share/keyrings/` or `/usr/share/keyrings/`. The sources file should also include a `signed-by` reference to the file.

Reply:
It should definitely go in `/usr/local/share/keyrings`. Everything in `/usr` (except `/usr/local`) should be managed entirely and solely by apt/dpkg, and nothing else.

Reply:
Also any solution we do document should not involve running `wget` as root.

Reply:
Does the new 'signed by' method also work for older distros or do we now need two sets of instructions?

Reply:
This should work for any Debian based distro.

Reply:
Thanks. For my own reference, the 'signed-by" syntax was apparently introduced in apt v1.3 (even 18.04 has apt v1.6) so we can just replace what we had and go with this method.

Reply:
Also, it should be clearer why /usr/local/share/keyrings/ over /usr/share/keyrings/. The former directory may not exist so you have to go and make it also. What a mess.